CMakeLists.txt

```
@@ -27,6 +27,7 @@ set(REALM_NO_TESTS)
 set(REALM_BUILD_LIB_ONLY)
 set(REALM_INSTALL_LIBEXECDIR)
 set(REALM_ENABLE_ENCRYPTION 1)
+set(REALM_ENABLE_EXPERIMENTAL)
```

```suggestion
add_compile_definitions(REALM_ENABLE_EXPERIMENTAL)
```
and probably move it next to the others